redbot/core/utils/__init__.py

```diff
@@ -20,7 +21,7 @@
     TYPE_CHECKING,
 )
 
-__all__ = ("bounded_gather", "deduplicate_iterables")
+__all__ = ("bounded_gather", "deduplicate_iterables", "AsyncGen")
```

Comment:
It's not a part of the PR, but could you add `bounded_gather_iter` to `__all__`?
```suggestion
__all__ = ("bounded_gather", "bounded_gather_iter", "deduplicate_iterables", "AsyncGen")
```